torch/distributed/_shard/sharded_optim/api.py

```diff
@@ -6,6 +7,15 @@
 
 
 class ShardedOptimizer(optim.Optimizer):
+    """
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.